app/lib/frontend/templates.dart

```diff
@@ -325,12 +379,43 @@ class TemplateService {
       'tabs': tabs,
       'has_no_file_tab': tabs.isEmpty,
       'version_count': '$totalNumberOfVersions',
+      'icons': LogoUrls.versionsTableIcons, // experimental_design: new
     };
-    return _renderPage(
-      'pkg/show',
-      values,
+    return values;
+  }
+
+  /// Renders the `views/private_keys/show.mustache` template.
```

Comment:
This comment seems wrong.

Reply:
Yeah, the original of the copy-paste was wrong too, fixed both.